content/resources/support/contributors-requirements.md

```diff
@@ -0,0 +1,62 @@
+---
+type: "page"
+title: "Contributor Requirements"
+subtitle: ""
+draft: false
+sidebar: true
+---
+
+{{< content-start  >}}
+
+# Contributor Requirements
+
+## Requirements for new contributors
+
+We welcome new contributors to our community - we want you to use QGIS as a platform for doing your work - and we believe that QGIS is a great platform for that. In order to contribute code, patches, or documentation to the QGIS project, all developers/team members must:
+
+- Abide by the [Social Contract](#social-contract).
+
+- Be familiar with the development process and the reference material available in the [Developers Guide for QGIS](https://docs.qgis.org/testing/en/docs/developers_guide/index.html), and the CODING document in the root of the QGIS Sources.
```

Comment:
It looks like there is no CODING document currently in the root of the QGIS Sources. The CODING document https://github.com/qgis/QGIS/blob/release-2_12/CODING was transferred long ago as a Website page and subsequently ad the Developers Guide for QGIS documentation page https://docs.qgis.org/testing/en/docs/developers_guide/index.html.
There is now a CONTRIBUTING document https://github.com/qgis/QGIS/blob/master/CONTRIBUTING.md in the root of the QGIS Sources.

I think it would be useful to inform the developer mailing list giving an overview of the current situation with the documents about development spread between website / docs and source code and proposed changes, so hopefully some other dev may have interesting suggestion to enhance the situation.

Reply:
Whatever it's easier to maintain: the important point is that we have those pages accessible somewhere ASAP or we cannot proceed with onboarding new core committers.

 These pages do not change frequently so I guess the website is ok.